app/gui2/src/util/containers.ts

```diff
@@ -80,7 +80,14 @@ export class MappedSet<T extends Object> {
    * The function should return values that are `===`-equal for values that should be considered equal.
    *
    */
-  constructor(private readonly valueMapper: (key: T) => any) {}
+  constructor(
+    private readonly valueMapper: (key: T) => any,
```

Comment:
Why is the return type `any` if the docs specify that it should map keys to strings?

Reply:
It does not have to return string. Updated docs.